docs/source/examples/demo_ml_commons_integration.ipynb

```diff
@@ -0,0 +1,402 @@
+    "        verify_certs=False\n",
+    "    )\n",
+    "    return client "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "89e1cb2a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = get_os_client()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4da9e0de",
+   "metadata": {},
+   "source": [
+    "## Step 1: Upload NLP model from local file to Opensearch cluster\n",
+    "\n",
+    "We can upload machine learning models to Opensearch cluster using MLCommons upload_model api. In this demo we will show how can we upload model which is stored in our local files. \n",
+    "\n",
+    "To demonstrate, we download the model zip file from the url: https://github.com/opensearch-project/ml-commons/raw/2.x/ml-algorithms/src/test/resources/org/opensearch/ml/engine/algorithms/text_embedding/all-MiniLM-L6-v2_torchscript_sentence-transformer.zip?raw=true\n",
+    "\n",
+    "To upload model to the cluster, we need a zip file containing a torchScript file (.pt extension) and a tokenizer.json file. Please refer to the previous download. We also need a json file with defining the config information with following these request fields: \n",
+    "\n",
+    "https://opensearch.org/docs/latest/ml-commons-plugin/api/#request-fields\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "c7b0ff7e",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total number of chunks 9\n",
+      "Sha1 value of the model file:  9376c2ebd7c83f99ec2526323786c348d2382e6d86576f750c89ea544d6bbb14\n",
+      "Model meta data was created successfully. Model Id:  jtuRoIUBqB81FWKil3qA\n",
+      "uploading chunk 1 of 9\n",
+      "Model id: {'status': 'Uploaded'}\n",
+      "uploading chunk 2 of 9\n",
+      "Model id: {'status': 'Uploaded'}\n",
+      "uploading chunk 3 of 9\n",
+      "Model id: {'status': 'Uploaded'}\n",
+      "uploading chunk 4 of 9\n",
+      "Model id: {'status': 'Uploaded'}\n",
+      "uploading chunk 5 of 9\n",
+      "Model id: {'status': 'Uploaded'}\n",
+      "uploading chunk 6 of 9\n",
+      "Model id: {'status': 'Uploaded'}\n",
+      "uploading chunk 7 of 9\n",
+      "Model id: {'status': 'Uploaded'}\n",
+      "uploading chunk 8 of 9\n",
+      "Model id: {'status': 'Uploaded'}\n",
+      "uploading chunk 9 of 9\n",
+      "Model id: {'status': 'Uploaded'}\n",
+      "Model uploaded successfully\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "'jtuRoIUBqB81FWKil3qA'"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "\n",
+    "model_path = '/Volumes/workplace/upload_content/all-MiniLM-L6-v2_torchscript_sentence-transformer.zip'\n",
+    "model_config_path = '/Volumes/workplace/upload_content/all-MiniLM-L6-v2_torchscript.json'\n",
```

Comment:
Can we paste this json file content here for easy reference?

Reply:
Yeah sure!!